packages/cli/src/util/telemetry/index.ts

```diff
@@ -157,8 +157,16 @@ export class TelemetryClient {
     return;
   }
 
-  trackFlagHelp() {
-    this.trackCliFlag('help');
+  trackCliFlagHelp(command: string, subcommands?: string | string[]) {
+    let subcommand;
+    if (subcommands) {
+      subcommand = Array.isArray(subcommands) ? subcommands[0] : subcommands;
```

Comment:
Did you find an instance of multiple subcommands?

Reply:
I didn't check, but the types claim they're allowed so I followed the current types.